raiden-dapp/src/components/overlays/InfoOverlay.vue

```diff
@@ -38,60 +40,77 @@ export default class InfoOverlay extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/dimensions';
 @import '@/scss/mixins';
 @import '@/scss/colors';
 
 .info-overlay {
-  align-items: center;
   background-color: $transfer-screen-bg-color;
   border-radius: 10px;
-  display: flex;
-  flex-direction: column;
-  height: calc(844px);
-  padding: 0 46px;
+  height: 844px;
```

Comment:
Why such an absolute value here and not relative to the parent (i.e. 100% like the width below)? I think this would even allow to unify it with the mobile view. Not too sure. But it look more complicated than it is...

Reply:
Since it is part of the header it would be relative to the header, the header is not the size of the application and it would therefore look like this:

<img width="650" alt="Screenshot 2021-04-23 at 11 57 37" src="https://user-images.githubusercontent.com/43838780/115855061-25396700-a42b-11eb-8be4-53d6a5030f79.png">

Reply:
Ah okay. Damn. We need to have this as a variable somewhere...